cpp/lib/src/AppArmorUtil.cc

```diff
@@ -75,9 +75,11 @@ ProfileMode GetProfileMode(const std::string &profile_id) {
 }
 
 
-void InstallProfile(const std::string &profile_path) {
-    static const std::string PROFILES_DIR("/etc/apparmor.d/local/");
-    const std::string target_path(PROFILES_DIR + FileUtil::GetBasename(profile_path));
+static const std::string PROFILES_DIR("/etc/apparmor.d/local");
+
+
+void InstallLocalProfile(const std::string &profile_path) {
+    const std::string target_path(PROFILES_DIR + "/" + FileUtil::GetBasename(profile_path));
     FileUtil::CopyOrDie(profile_path, target_path);
```

Comment:
Is it safe to always overwrite the installed profile, if there is one?  Should we add an additional boolean function parameter called "overwrite" set to true by default?

Reply:
As far as i know, this is safe and the new profile will just replace the old one. CopyOrDie will just replace the file without error, and AppArmor will use the new settings. Since CopyOrDie does not support and overwrite parameter, i guess it's not necessary to introduce an overwrite parameter here either.